wod_predictor/wod_predictor/WODFeatureExtraction/features_extracted1.csv

```diff
@@ -0,0 +1,34 @@
+Repetition,Time,Format,Equipment,ExerciseType,TargetMuscle,Difficulty
+150, undefined, rounds for time, dumbbell and box, snatches and burpee box jump-overs, full body, low
+-1, 12 minutes, AMRAP, dumbbells and pull-up bar and barbell, weighted walking lunge and toes-to-bars and power cleans and bar muscle-ups, undefined, medium
+-1,24:00,rounds for time,barbell,pull-ups and squat snatches,back and shoulders and legs,high
+-1,13 minutes,AMRAP,barbell and wall ball and rowing machine,deadlifts and wall-ball shots and rowing and handstand push-ups,undefined,high
+90, undefined, rounds for time, barbell and jump rope, thrusters and double-unders, undefined, low
+-1,20 minutes,AMRAP,dumbbell and machine,toes-to-bars and dumbbell hang clean and jerks and row,core and shoulders and back,medium
+55, undefined, rounds for time, dumbbell, dumbbell squats and bar-facing burpees, undefined, medium
+-1, undefined, undefined, dumbbell and barbell, squat and burpees and clean, undefined, high
+488, undefined, rounds for time, jump rope and barbell and rings and dumbbell, double-unders and overhead squats and ring muscle-ups and dumbbell snatches and bar muscle-ups, undefined, high
+126, undefined, rounds for time, barbell, deadlifts and handstand push-ups and handstand walk, undefined, medium
+-1,undefined,rounds for time,barbell,pull-ups and thrusters,upper body and lower body,medium
+-1,15 minutes,AMRAP,wall-ball and rowing machine,wall-ball shots and rowing,undefined,low
+-1,20,AMRAP,barbell,jump rope,gymnastics,weightlifting,core,legs,undefined
```

Comment:
This line raised an error while reading in. I guess for target muscle, the LLM replied "core,legs", buts saving as a csv would put these as separate colums. We'd want to restrict the answers available as well. Since some might be "core and shoulders", and another is "shoulders and legs and core". Might be good to have some standard formatting. Later on we can combine it by categorical encoding this column (`target_muscle_core`, `target_muscle_legs`, etc).